Rb.Forms.Barcode.Droid/View/CameraService.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using Rb.Forms.Barcode.Pcl;
+using Rb.Forms.Barcode.Pcl.Logger;
+using Rb.Forms.Barcode.Droid.Camera;
+using Rb.Forms.Barcode.Droid.Logger;
+using Android.Views;
+using Xamarin.Forms;
+
+namespace Rb.Forms.Barcode.Droid.View
+{
+    public class CameraService : ILog
+    {
+        private readonly BarcodeScanner renderer;
+
+        private readonly ScannerCamera scannerCamera;
+        private readonly AutoFocusCallback autoFocus;
+
+        public CameraService(BarcodeScanner renderer, ScannerCamera scannerCamera)
+        {
+            this.renderer = renderer;
+            this.scannerCamera = scannerCamera;
+            this.autoFocus = new AutoFocusCallback(scannerCamera);
+        }
+
+        public void OpenCamera(ISurfaceHolder holder)
+        {
+            try {
+                scannerCamera.OpenCamera();
+                scannerCamera.AssignPreview(holder);
+
+                renderer.OnCameraOpened();
+
+                renderer.PreviewActive = true;
+            } catch (Exception ex) {
+                this.Debug("Unable to open camera");
+                this.Debug(ex.ToString());
+            }
+        }
+
+        public void ReleaseCamera()
+        {
+            autoFocus.Enabled = false;
+
```

Comment:
haben die vielen leerzeilen in der metode eine bedeutung. möchtest du die calls besonders hervorheben?